src/module-1/cutStrings/index.js

```diff
@@ -1 +1,14 @@
-export const cutStrings = (arr = []) => {};
+export const cutStrings = (arr = []) => {
+  if (arr.length > 0) {
+    let shortestStr = arr.reduce((a, b) => a.length <= b.length ? a : b);
+    let result = [];
+
+    arr.forEach(function (value) {
```

Comment:
use arrow func here - `value => {}` for you use them already in other places. Let's keep consistency here